CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
       - ESDT system smart contract
     - Added `BalanceBuilder` interface
     - Added `NativeSerializer`
+    - Added `ExtensionProvider`
```

Comment:
This is actually added in the `Contract Wrapper` PR section.
We should add a new section, under everything from `Fixed ledger login`:

```
- [Fixed Ledger login](https://github.com/ElrondNetwork/elrond-sdk-erdjs/pull/48)
   - Fixed Ledger login by also setting the index to the device.
  
   - ### Breaking changes:
     - Removed `addressIndex` from constructor
     - Use `addressIndex` from `options` object of `login` function


- [Added Extension Provider for dapp integration #59](https://github.com/ElrondNetwork/elrond-sdk-erdjs/pull/59)
```

Or something like that